charts/jx3/jx-pipelines-visualizer/values.yaml.gotmpl

```diff
@@ -26,6 +26,8 @@ serviceAccount:
   annotations:
 {{- if eq .Values.jxRequirements.cluster.provider "gke" }}
     iam.gke.io/gcp-service-account: {{ .Values.jxRequirements.cluster.clusterName }}-jxui@{{ .Values.jxRequirements.cluster.project }}.iam.gserviceaccount.com
+{{- else if eq .Values.jxRequirements.cluster.provider "eks" }}
+    eks.amazonaws.com/role-arn: arn:aws:iam::{{ .Values.jxRequirements.cluster.project }}:role/{{ .Values.jxRequirements.cluster.clusterName }}-jx-pipelines-visualizer
```

Comment:
What policies need to be attached for this role? Would be nice to add them to the tf module.

Reply:
Read access to log storage bucket I think